ember-cli-build.js

```diff
@@ -18,11 +22,11 @@ module.exports = function(defaults) {
   var demoAppCssTree = new BroccoliSass(['src/demo-app'], './demo-app.scss', 'demo-app/demo-app.css');
   var demoCssTree = getCssTree('demo-app');
   var componentCssTree = getCssTree('components');
-  var angularAppTree = new Angular2App(defaults);
+  var angularAppTree = new Angular2App(defaults).toTree();
```

Comment:
It returns a Broccoli tree. It was called below before, but in an iteration I needed access to it. I'll undo that.